### PR TITLE
Moving global sysctl settings to where they belong

### PR DESCRIPTION
**Pull request by rvs**

### CODE REVIEW

File: pkg/pillar/cmd/zedrouter/flowstats.go [hunk 1]
```diff
@@ -855,86 +854,3 @@ func bridgeStrToNum(bnStr string) (int, error) {
 	}
 	return bnNum, nil
 }
-
-func flowTimeOutSet(item string, origValue int) {
-
-	tOutValue := origValue + int(timeoutSec) // add 150 seconds
```
Comment (eriknordmark) [inline pkg/pillar/cmd/zedrouter/flowstats.go]: This key part of the logic has gone missing; a comment saying "update" isn't the same as specifying the relationship between the timer values i.e., that timeoutSec needs to be added to a particular subset of the sysctl values.

Comment (rvs) [reply pkg/pillar/cmd/zedrouter/flowstats.go]: Can you suggest a better comment? Note that I would agree with you if you were reading the current values and adding 150 to them. The way it is implement -- is no better or worse compared to what this PR is doing.

Comment (eriknordmark) [reply pkg/pillar/cmd/zedrouter/flowstats.go]: @naiming-zededa is the one that came up with the scheme where we need to add that constant. Can you check with him if he has suggestions on how to handle this?

Comment (rvs) [reply pkg/pillar/cmd/zedrouter/flowstats.go]: Let me check with @naiming-zededa today (now that I have a bit more spare time). Additionally, my interest in pulling all this into a separate container that was specifically designed to handle sysctl setting is that:
   1. it has higher chance of working out of the box (compared to us trying to tweak pillar container settings, etc.)
   2. if it doesn't -- we have the linuxkit community to help us figure out why not

At the same time I agree that some of the more involved logic (like the one for StrongSWAN) is not easy to be converted into this style -- so we'll punt on those for now. It is just that this one is a borderline that I feel like we can pull into a specially designed container.

**Comment by rvs** [comment]
In today's installment of: fascinating stuff I find when I dig into the guts of linuxkit it turns out that EVE has been ignoring Alpine level sysctl defaults for all this time. Why -- well, because it wasn't properly bind mounted. So now I'm doing /etc/sysctl.d:/etc/sysctl.d but I also pulled the linuxkit defaults into our repo so we can have a central place to reconcile it all.

This @eriknordmark solves the mystery. The only part that we may still want to do is to pull alpine defaults into our repo as well, but perhaps that would be going too far: https://git.alpinelinux.org/aports/tree/main/alpine-baselayout/APKBUILD#n158

**Comment by rvs** [comment]
Ok, since @naiming-zededa approved it -- merging it in now.

File: pkg/dom0-ztools/rootfs/etc/sysctl.d/01-linuxkit.conf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# settings from https://raw.githubusercontent.com/linuxkit/linuxkit/master/pkg/sysctl/etc/sysctl.d/00-linuxkit.conf
+# from Alpine defaults
+net.ipv4.tcp_syncookies = 1
+net.ipv4.conf.default.rp_filter = 1
+net.ipv4.conf.all.rp_filter = 1
+net.ipv4.ping_group_range=999 59999
+# general limits
+vm.max_map_count = 262144
+vm.overcommit_memory = 1
```
Comment (eriknordmark) [inline pkg/dom0-ztools/rootfs/etc/sysctl.d/01-linuxkit.conf]: What I can't sell what I don't own? ;-)

File: pkg/dom0-ztools/rootfs/etc/sysctl.d/02-eve.conf [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# NOTE that EVE still has dynamic sysctl settings under
+# pkg/pillar/cmd/zedrouter/{ipsec,strongswan}.go
+#
+# Make sure to reconcile those when updating this file
+
+# global kernel settings
+kernel.softlockup_panic = 1
```
Comment (eriknordmark) [inline pkg/dom0-ztools/rootfs/etc/sysctl.d/02-eve.conf]: This one is problematic on Hikey - hsysctl -w angs when setting it.
Does sysctl.d have a way to have conditional (on arch etc) files?

**Pull request merged**